models/classes/dataBinding/class.GenerisInstanceDataBinder.php

```diff
@@ -196,7 +196,7 @@ private function isBlockedForModification(core_kernel_classes_Property $property
             return $property->isStatistical();
         }
 
-        return false;
+        return $property->isWritable();
```

Comment:
```suggestion
        return !$property->isWritable();
```
Maybe we should return false if it's writable as the method is `isBlockedForModification`. Because we should block for writing if a property is not writable

Reply:
Good point, seems I forgot to change it while worked on unit tests